opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -1052,6 +1052,7 @@ namespace Opm
             };
             forAllGroupData(computeSize);
             sz += injection_group_vrep_rates.size();
+            sz += current_alq_.size();
```

Comment:
You might want to check your assumption (equal size of the map on all proceses) at least when compiled without `-DNDEBUG`